src/jlgen.jl

```diff
@@ -624,7 +624,7 @@ function compile_method_instance(@nospecialize(job::CompilerJob))
     if VERSION < v"1.12.0-DEV.1667"
         cgparams = (; lookup = Base.unsafe_convert(Ptr{Nothing}, lookup_cb), cgparams... )
     end
-    if VERSION > v"1.13.0-DEV.285"
+    if VERSION >= v"1.12.0-DEV.2126"
```

Comment:
```suggestion
    if v"1.12.0-DEV.2126" <= VERSION < v"1.13-" || VERSION >= v"1.13.0-DEV.285"
```

We need to be really accurate for bisection purposes.